src/components/search/Search.jsx

```diff
@@ -25,18 +26,43 @@ class Search extends Component {
     const { searchData } = this.props;
     const searchItems = searchData.filter(edge => Search.searchFor(edge, searchQuery));
     const searchResultItems = searchItems ? searchItems.map(edge => ArtistCard(edge)) : null;
+    const searchLanguage = searchData.map(edge => edge.node.lang);
+    const currentLanguage = searchLanguage[0];
+
+    let placeholderText;
```

Comment:
variable is must not be empty
let placeholderText = null;

Reply:
fixed: 01e5535629c1311aa8096784fa7f140a585256b3